src/Data/VerifiedEq/Instances/Contra.hs

```diff
@@ -1,13 +1,13 @@
-{-@ LIQUID "--higherorder"        @-}
-{-@ LIQUID "--automatic-instances=liquidinstances" @-}
+{-@ LIQUID "--reflection" @-}
+{-@ LIQUID "--ple"        @-}
 
 module Data.VerifiedEq.Instances.Contra (veqContra, eqContra) where
 
 import Data.Functor.Contravariant
 import Data.VerifiedEq
 import Language.Haskell.Liquid.ProofCombinators
 
-{-@ axiomatize eqContra @-}
+{-@ reflect eqContra @-}
```

Comment:
I'm glad that we're finally calling this thing `reflect` like in the literature :)